src/lib.rs

```diff
@@ -513,13 +514,15 @@ impl TaskLocals {
     pub fn context<'p>(&self, py: Python<'p>) -> Bound<'p, PyAny> {
         self.context.clone_ref(py).into_bound(py)
     }
+}
 
+impl Clone for TaskLocals {
     /// Create a clone of the TaskLocals by incrementing the reference counters of the event loop and
     /// contextvars.
-    pub fn clone_ref(&self, py: Python<'_>) -> Self {
+    fn clone(&self) -> Self {
         Self {
-            event_loop: self.event_loop.clone_ref(py),
-            context: self.context.clone_ref(py),
```

Comment:
I don't have merge/publish access, so I don't make the end decision, but one thing I was wondering: usually this crate is just released whenever there's a breaking pyo3 release. But if you want a release before then, I think it would help to make this PR non-breaking. We could keep this `clone_ref` method as a dummy that just calls `self.clone()`, add a `#[deprecated]`, and remove it in the next breaking release. (I'm not sure if it's ok to add `#[deprecated]` in a patch release?)